src/synced_services/redmine/redmine_synced_service.ts

```diff
@@ -643,6 +673,12 @@ export class RedmineSyncedService implements SyncedService {
       )
 
       const updated = await this.getTimeEntryById(originalTimeEntry.originalEntry.id);
+      //if project not in mappings resync just in case
+      if (updated && (!project || updated.projectId != project.id)) {
+        const date = new Date(updated.lastUpdated);
+        updated.lastUpdated = new Date(date.getTime());
+        updated.lastUpdated.setDate(date.getDate() - 1);
```

Comment:
Aky je dovod? Nerobi to Redmine?

Reply:
Tohle je úprava T2T lastUpdated aby byl starší než poslední změna a provedl se back sync. Normálně se nastavuje na last_updated dané entity. Tímto forcuji že nastala změna, která se má propsat. Je to trochu hack.

Reply:
Okj, dava zmysel, vdaka za vysvetlenie